backend/flask_api.py

```diff
@@ -22,43 +22,40 @@
 from pymongo import MongoClient, ReturnDocument
 from werkzeug.utils import secure_filename
 from werkzeug.routing import BaseConverter
-from mongoengine.fields import UUIDField
+from mongoengine.fields import UUIDField, ListField, StringField, BooleanField
 from bson.json_util import dumps
 
 import uploader
 from apiexception import ApiException
 
 
-es = Elasticsearch([{'host': 'elasticsearch', 'port': 9200}])
-client = MongoClient('mongodb:27017')
-db = client.knexDB
-coll = db.projects
-app = Flask(__name__)
+# Create app
+app = Flask(__name__, static_url_path='')
```

Comment:
If of interested, this basically means, that static files should be served from the root of the webside, instead of "/static/" as is default.